.github/workflows/cmake.yml

```diff
@@ -21,6 +21,23 @@ jobs:
     - uses: actions/checkout@v3
       with:
          submodules: 'recursive'
+         fetch-depth: '0'
+    - run: |
+        git fetch --prune --tags
+        git tag --list
+        git fetch --all --tags
```

Comment:
what's the purpose of double git-fetch? what's the purpose of --prune, since it's a fresh checkout I don't think it's needed? Also, wouldn't it get all of the recent tags anyway when it's cloning? fetch iirc only needed if you have an older repo and just doing git-pulls

Reply:
Is now simplified